pkg/retriever/bitswapretriever_test.go

```diff
@@ -161,7 +174,11 @@ func TestBitswapRetriever(t *testing.T) {
 				cid1: {types.StartedCode, types.FirstByteCode, types.SuccessCode},
 				cid2: {types.StartedCode, types.FirstByteCode, types.SuccessCode},
 			},
-			expectedCids: append(tbc1Cids[:5], tbc2Cids[:5]...),
+			expectedCids: append(append([]cid.Cid{}, tbc1Cids[:5]...), tbc2Cids[:5]...),
```

Comment:
I spent ~2 hours trying to figure out a series of bugs that arose from this ... tbc1Cids was getting modified by the setup of this test case, causing unrelated things to fail, sending me down the wrong rabbit holes! Similar problem in the previous case too.